src/hpguppi_ata_blade_obs_rawdisk_thread.c

```diff
@@ -358,36 +373,66 @@ static void *run(hashpipe_thread_args_t * args)
         if(directio) {
           open_flags |= O_DIRECT;
         }
-        for(i = 0; i < nbeams; i++){
-          sprintf(fname, "%s-beam%04d.%04d.raw", pf.basefilename, i, filenum);
+        if(nbeams == 0){
+          sprintf(fname, "%s-beam.raw", pf.basefilename);
```

Comment:
remove the `-beam` suffix, include the `.%04d.` section, as per [L313](https://github.com/MydonSolutions/hpguppi_daq/blob/seti-ata-8bit/src/hpguppi_atasnap_obs_rawdisk_thread.c#L313)